### ISSUE TITLE
sprint to finalize this repository?

### ISSUE THREAD

**Issue by davidedelvento** [swcarpentry/hpc-novice#38]
Folks,
I'm leading the organization of [this conference](https://sea.ucar.edu/conference/2018) and as part of it we would like to have some smallish "getting-things-done" 1-day sessions. 
Would people be interested in participating (in person) in such a session?
Thanks,
Davide

**Comment by shwina** [comment]
I would certainly volunteer to participate remotely, but unfortunately cannot attend in person

**Comment by davidedelvento** [comment]
@shwina If we have a critical mass of people attending in person, I guess it'll be fine for a handful to be remote. However if the whole participant list can only be remote, then there's not point on piggybacking it on the conference.

**Comment by psteinb** [comment]
Same here! As I am from overseas, I'd appreciate a remote connection. What did you have in mind specifically?

**Comment by davidedelvento** [comment]
I had in mind reviewing the whole repositories and make sure the lessons are (as close as possible to) being ready for being taught to students the day after this event.

**Comment by ChristinaLK** [comment]
@davidedelvento, I know of someone else who was tentatively planning an hpc-novice like sprint at a different event.  Not sure if it was going to be at this repo though.  Do you want me to connect you with them, just to see if there's any points where you could collaborate?

**Comment by davidedelvento** [comment]
@ChristinaLK yes please put me in touch with them. My contacts (in case you think an off-github conversation works best) are at https://staff.ucar.edu/users/ddvento

**Comment by r4space** [comment]
I missed the SC BOF - did this get discussed at all?  I might be interested if it happens

**Comment by davidedelvento** [comment]
I did not attend the supercomputing BOF either.
As far as I know the sprint is not happening at the SEA conference for lack of critical mass...

**Comment by dbrunson** [comment]
Here's the page for the BoF that has links to everything: https://hpc-uk.github.io/sc17-hpccarpentry-bof/   It was a GREAT BoF!  Kudos to the organizers and everyone that participated.

**Comment by psteinb** [comment]
Thanks for sharing the link. Related to this, I am wondering if anyone 
would be interested in a similar BoF at ISC2018 next year? Please drop a 
+1 to this comment, if you think such a thing is useful!

**Comment by shwina** [comment]
Thanks, @psteinb, while I will likely not be able to attend, I certainly think it's an excellent idea

**Comment by statkclee** [comment]
+1
